.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
   directories:
     - vendor
     - $HOME/.composer/cache
+    - $HOME/.phpbrew
```

Comment:
I don't feel too strongly about this, but because there are already phpunit shims injected by Travis, I'd rather only have this around when we actually need it, so that anyone who needs to debug this in the future has a slightly easier time figuring out what phpunit is actually getting run.